qcfractal/interface/collections/collection.py

```diff
@@ -581,8 +581,15 @@ def status(
                 for spec in specs:
                     list_specs.append(self.query(spec))
 
+            def get_status(item):
+                try:
+                    return item.status.value
+                except:
+                    return None
+
             # apply status by column then by row
-            df = self.df[list_specs].apply(lambda col: col.apply(lambda entry: entry.status.value))
+            df = self.df[list_specs].apply(lambda col: col.apply(get_status))
```

Comment:
`lambda entry: entry.status.__dict__.get("value", None)`?

Reply:
Doesn't work when `entry` is NaN.